include/fluent-bit/flb_intermediate_metric.h

```diff
@@ -0,0 +1,23 @@
+// Fluent Bit intermediate metric representation 
```

Comment:
- append Fluent Bit licensing comments: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#licensing

Reply:
every header file content must be inside a conditional macro, e.g:

```c
#ifndef FLB_INTERMEDIATE_METRIC
#define FLB_INTERMEDIATE_METRIC
...
#endif

note: I am not confident the name of this file is the proper one
```

Reply:
Updated.